.babelrc

```diff
@@ -1,7 +1,16 @@
 {
-    "presets": ["./.babelrc.js"],
```

Comment:
Is there a reason why we can't use presets from local JS file? Because that will break linking of this package.

Reply:
Nope. I'll add it back!